yorkie/src/main/kotlin/dev/yorkie/document/crdt/RgaTreeList.kt

```diff
@@ -0,0 +1,242 @@
+package dev.yorkie.document.crdt
+
+import androidx.annotation.VisibleForTesting
```

Comment:
We can avoid dependency on Android here.

```suggestion
import com.google.common.annotations.VisibleForTesting
```